data/sql/updates/db_world/2016_12_07_00.sql

```diff
@@ -0,0 +1,30 @@
+-- More Blizzlike Waypoint for npc_black_knight_skeletal_gryphon, NPC= 35491
+-- Gryphon updated waypoint corrected the orientation when landing.
+-- Data is DELETED and RE-INSERTED, becouse new waypoint has new 5 blizzlike waypoint.
```

Comment:
a note for future:  just use the PR description for this kind of comments, there is not need to put in the actual SQL file